scraper.rb

```diff
@@ -41,4 +47,4 @@ def scrape_page(page)
   page = agent.get(url)
   puts "Scraping page #{no} of " + number_pages.to_s + "..."
   scrape_page(page)
-end
+end
```

Comment:
Getting 'no newline at end of file' on this see https://robots.thoughtbot.com/no-newline-at-end-of-file